controllers/apply-next/simple/fields.test.js

```diff
@@ -1,36 +0,0 @@
-/* eslint-env jest */
```

Comment:
Removing this test as it tests at an awkward level, requiring another unit in order to be able to test the current one suggests that this test should be at a different level.